tests/testthat/test-param-estimates.R

```diff
@@ -2,21 +2,24 @@ context("Test param_estimates functions")
 
 skip_if_not_drone_or_metworx("test-param-estimates")
 
-# setup and teardown
-cleanup()
-withr::defer(cleanup())
 
 # build reference
 ref_df1 <- dget(PARAM_REF_FILE)
 
 withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
 
   test_that("param_estimates.bbi_model_summary gets expected table [BBR-PEST-001]", {
+    cleanup()
+    withr::defer(cleanup())
```

Comment:
Fine as is, but a thought: you could make the `.f` arg of `clean_test_enviroment` optional.  Then this `cleanup() / withr::defer(cleanup())` pair that's repeated a few times could be reduced to `clean_test_enviroment()`.